tests/FilterTest.php

```diff
@@ -91,4 +91,11 @@ public function isSmallString($str) {
     public function test_named_constant() {
         $this->assertTrue(is_callable(f\filter));
     }
+
+    /**
+     * @test
+     */
+    public function should_work_like_array_filter_with_callback_null() {
+        $this->assertEquals([], f\filter(f\id, [0, null, false, '', '0']));
```

Comment:
Heb je dit goed getest @martijngastkemper ? Want ik mis de haakjes bij `f\id()`. Volgens mij gaat die nu op zoek naar een constante `id`.

Reply:
Since #28 all functions are available as constants. `f\id()` also works, because it's uses `autocurry`. I've updated the test to make sure you aren't right 😉

Reply:
Nice!!!